api/segment_membership/services.py

```
@@ -153,6 +153,15 @@ def compute_segment_counts_for_project(
                 reason="untranslatable",
             )
             continue
+        select_clauses.append(
```

Ruff (0.15.21)

[error] Undefined name `select_clauses`

(F821)